Tools/setup/ubuntu.sh

```diff
@@ -146,7 +146,7 @@ if [ $(which arm-none-eabi-gcc) ]; then
 	GCC_FOUND_VER=$(echo $GCC_VER_STR | grep -c "${NUTTX_GCC_VERSION}")
 fi
 
-	if [ ! ${GCC_FOUND_VER+x} && $GCC_FOUND_VER -eq "1" ]; then
+	if [[ ! -z $GCC_FOUND_VER && $GCC_FOUND_VER -eq "1" ]]; then
```

Comment:
Why not just `if [ "$GCC_FOUND_VER" = "1" ]; then`?

Reply:
Cause I don't know bash properly. Feel free to fix :smile:

Reply:
https://github.com/PX4/Firmware/pull/14935